### ISSUE TITLE
Interactive demo doesn't seem to be accessible

### ISSUE THREAD

**Issue by Zireael07** [pixijs/examples#111]
I read on the main page that all pixi content is accessible. From a brief look through the accessibility code here on GitHub, it seems content should become accessible (i.e. generate matching HTML) on pressing TAB. However, when I press tab on the examples page https://pixijs.io/examples/#/interaction/interactivity.js, nothing happens.

**Comment by bigtimebuddy** [comment]
Adding the follow should be a simple fix to enable accessibility for this (and other demos).
```js
button.accessible = true;
button.accessibleTitle = 'Button ' + (i + 1);
```

**Comment by Zireael07** [comment]
Ah, so it's not automatic? Main page should be updated then, it makes it sound like it's automatic.

**Comment by bigtimebuddy** [comment]
By main page, I assume you mean pixijs.com. here's the copy about this feature:

> Pixi is an inclusive technology and all content can be made to be screen reader accessible with ease. The only WebGL renderer out there that does.

I think "can be made" describe this feature accurately.
